tfhe/src/integer/client_key/radix.rs

```diff
@@ -131,6 +134,15 @@ impl RadixClientKey {
     pub fn num_blocks(&self) -> usize {
         self.num_blocks
     }
+
+    pub fn new_compressed_compression_decompression_keys(
+        &self,
+        private_compression_key: &CompressionPrivateKeys,
+    ) -> (CompressedCompressionKey, CompressedDecompressionKey) {
+        self.key
+            .key
+            .new_compressed_compression_decompression_keys(private_compression_key)
+    }
```

Comment:
just thought about this does a client_key.as_ref().as_ref() allow to get access to the shortint key to generate the compression keys ? in any case @mayeul-zama was working to add the missing keys in the integer API level so this will likely be refactored fairly quickly

Reply:
I'm not sure. I wasn't able to use it like that. Not sure how to solve this. `cks` has `integer::ClientKey`. What should be the type of the first `as_ref`?

```
error[E0282]: type annotations needed
   --> tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs:159:21
    |
159 |         let x = cks.as_ref().as_ref().new_compressed_compression_decompression_keys
    |                     ^^^^^^   ------ type must be known at this point
```

Reply:
yeah ok, I think I know why, it's fine to keep the above though